### PR TITLE
ceph-iscsi: fix certificates generation

### PR DESCRIPTION
**Pull request by leseb**
Prior to this patch, the certificates where being generated on a single
node only (because of the run_once: true). Thus certificates were not
distributed on all the gateway nodes.

This would require a second ansible run to work. This patches fix the
creation and keys's distribution on all the nodes.

Closes: https://bugzilla.redhat.com/show_bug.cgi?id=1540845
Signed-off-by: Sébastien Han <seb@redhat.com>

### CODE REVIEW

**Comment by leseb** [comment]
jenkins test luminous-ansible2.4-centos7_cluster

**Comment by leseb** [comment]
jenkins test luminous-ansible2.4-centos7_cluster

**Comment by leseb** [comment]
jenkins test luminous-ansible2.4-centos7_cluster

**Comment by leseb** [comment]
jenkins test luminous-ansible2.4-centos7_cluster

**Comment by leseb** [comment]
@guits there is no need to run the CI more than this scenario.

File: group_vars/iscsi-gws.yml.sample [hunk 1]
```diff
@@ -58,3 +58,6 @@ dummy:
 
 #client_connections: {}
 
+# Wether or not to generate secure certificate to gateway nodes
```
Comment (ktdreyer) [inline group_vars/iscsi-gws.yml.sample]: Wether -> Whether.

Do you want to clarify "gateway nodes" is "iscsi gateway nodes"? (not rados gateway)

Comment (leseb) [reply group_vars/iscsi-gws.yml.sample]: Done, thanks!

**Comment by leseb** [comment]
jenkins test luminous-ansible2.4-centos7_cluster

**Pull request merged**